src/bin/js/dune

```diff
@@ -0,0 +1,43 @@
+(documentation
+  (package alt-ergo)
+  (mld_files :standard)
+)
+
+(executable
+  (name         main_text_js)
+  (libraries    common zarith_stubs_js js_of_ocaml-compiler)
+  (modules      main_text_js)
+  (modes        js)
+  (js_of_ocaml
+    (flags --no-source-map --linkall --disable genprim)
+  )
+)
+
+(library
+  (name options_rec)
+  (modules      options_rec)
+)
+
+(executable
+  (name         worker_js)
+  (libraries    options_rec alt-ergo-lib alt-ergo-parsers common zarith_stubs_js js_of_ocaml js_of_ocaml-compiler lwt js_of_ocaml-lwt
+)
+  (modules      worker_js)
+  (modes        js)
+  (preprocess (pps js_of_ocaml-ppx lwt_ppx))
+  (js_of_ocaml
+    (flags --no-source-map --linkall --disable genprim)
```

Comment:
What are the reason for passing the two flags `--linkall` and `--disable genprim` ?

Reply:
None, it's old code from tests